components/RefinementList.js

```diff
@@ -43,6 +45,7 @@ class RefinementList extends React.Component {
     var rootClass = cx(this.props.cssClasses.root);
     var listClass = cx(this.props.cssClasses.list);
     var itemClass = cx(this.props.cssClasses.item);
+    var transformData = this.props.transformData || identity;
 
```

Comment:
You could check that its a function (simple typeof ..) and then data = transformData

Bc here we do iterate over data even ifs theres no transform to be done, works but feels unnecessary?